src/containers/org/OrgObjectPermissionsContainer.js

```diff
@@ -0,0 +1,151 @@
+/*
+ * @flow
+ */
+
+import React, { useEffect, useMemo, useState } from 'react';
+
+import styled from 'styled-components';
+import { List } from 'immutable';
+import { Types } from 'lattice';
+import {
+  AppContentWrapper,
+  CheckboxSelect,
+  SearchInput,
+  Typography,
+} from 'lattice-ui-kit';
+import { useRequestState } from 'lattice-utils';
+import { useDispatch, useSelector } from 'react-redux';
+import { RequestStates } from 'redux-reqseq';
+import type {
+  Ace,
+  Organization,
+  PermissionType,
+  UUID,
+} from 'lattice';
+import type { RequestState } from 'redux-reqseq';
+
+import {
+  ActionsGrid,
+  CrumbItem,
+  CrumbLink,
+  Crumbs,
+  Divider,
+  PlusButton,
+  Spinner,
+  StackGrid,
+} from '../../components';
+import { GET_ORGANIZATION_OBJECT_PERMISSIONS, getOrganizationObjectPermissions } from '../../core/permissions/actions';
+import { resetRequestState } from '../../core/redux/actions';
+import { PERMISSIONS } from '../../core/redux/constants';
+import { selectObjectPermissions, selectOrganization } from '../../core/redux/selectors';
+import { Routes } from '../../core/router';
+import { ObjectPermissionsCardStack } from '../permissions';
+import type { ReactSelectOption } from '../../types';
+
+const { PermissionTypes } = Types;
+
+const PERMISSION_TYPE_OPTIONS = [
+  { label: PermissionTypes.OWNER.toLowerCase(), value: PermissionTypes.OWNER },
+  { label: PermissionTypes.READ.toLowerCase(), value: PermissionTypes.READ },
+  { label: PermissionTypes.WRITE.toLowerCase(), value: PermissionTypes.WRITE },
+  { label: PermissionTypes.LINK.toLowerCase(), value: PermissionTypes.LINK },
+  { label: PermissionTypes.MATERIALIZE.toLowerCase(), value: PermissionTypes.MATERIALIZE },
```

Comment:
Isn't there also discover?

Reply:
We're ignoring discover:

https://jira.openlattice.com/browse/LATTICE-2578
https://jira.openlattice.com/browse/APPS-2381